src/Context/LoginContext.php

```diff
@@ -80,6 +81,45 @@ public function stepIAmNotLoggedIn()
      * @param string $password
      */
     public function stepILogInWith($email, $password)
+    {
+        $this->loginWith($email, $password);
+
+        // Check if MFA module is installed
+        if (!class_exists(RegisteredMethod::class)) {
+            return;
+        }
+
+        // Skip MFA registration if MFA module installed
+        $this->getMainContext()->getSession()->wait(300);
+        $page = $this->getMainContext()->getSession()->getPage();
+        $mfa = $page->find('css', '#mfa-app');
+        if (!$mfa) {
+            return;
+        }
```

Comment:
Is the expectation that you could get to this point and the login might have been successful? If not, then we should fail the scenario and provide a useful error message.

```suggestion
        assertNotNull($mfa, 'MFA is installed, but could not find the MFA form.');
```

Reply:
I don't think so, if you've already entered MFA for a user then you won't see the mfa form